packages/editor/src/client/blocks/richTextBlock/editor/elements.ts

```diff
@@ -10,7 +10,7 @@ export interface TableCell extends SlateElement {
   borderColor: string
 }
 
-export const defaultBorderColor: TableCell['borderColor'] = 'black'
+export const defaultBorderColor: TableCell['borderColor'] = '#000000'
```

Comment:
`export const DEFAULT_BORDER_COLOR`

Reply:
@roosnic1  maybe we need a naming conventions' list for different variables.